newIDE/app/src/CompactPropertiesEditor/index.js

```diff
@@ -229,8 +228,6 @@ const styles = {
   },
   container: { flex: 1, minWidth: 0 },
   separator: {
-    marginRight: -marginsSize,
```

Comment:
this was causing a nasty horizontal scroll

Reply:
Good catch

Reply:
double checked different objets panels, and can't see any issue